sample/src/main/java/com/canhub/cropper/sample/camera_java/presenter/SCameraPresenterJava.java

```diff
@@ -0,0 +1,169 @@
+package com.canhub.cropper.sample.camera_java.presenter;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.os.Build;
+import android.util.Log;
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.ContextCompat;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
+import com.canhub.cropper.CropImage;
+import com.canhub.cropper.sample.camera_java.app.SCameraFragmentJava;
+import com.canhub.cropper.sample.camera_java.domain.CameraEnumDomainJava;
+import com.canhub.cropper.sample.camera_java.domain.SCameraContractJava;
+import com.canhub.cropper.sample.camera_java.app.SCameraFragmentJava;
```

Comment:
Some imports here are not need anymore, please remember to clean and organise the imports.

On Mac you can do automatically with `control+option+o`